editor/import/resource_importer_scene.cpp

```diff
@@ -1245,6 +1245,10 @@ Node *ResourceImporterScene::_post_fix_node(Node *p_node, Node *p_root, HashMap<
 								col->set_owner(p_node->get_owner());
 								col->set_transform(get_collision_shapes_transform(node_settings));
 								col->set_position(p_applied_root_scale * col->get_position());
+								const Ref<PhysicsMaterial> &pmo = node_settings["physics/physics_material_override"];
+								if (!pmo.is_null()) {
```

Comment:
This could be `if (pmo.is_valid())` (same applies to the other cases), but no biggie.

https://github.com/godotengine/godot/blob/202e4b2c1e7f8b25738b93d0e4d5066453d3edf3/core/object/ref_counted.h#L201-L202